packages/f-registration/package.json

```diff
@@ -71,6 +66,11 @@
     "@vue/test-utils": "1.1.1",
     "axios-mock-adapter": "1.19.0",
     "flush-promises": "1.0.2",
-    "node-sass-magic-importer": "5.3.2"
+    "node-sass-magic-importer": "5.3.2",
+    "@justeat/f-button": "0.4.1",
+    "@justeat/f-card": "0.7.0",
+    "@justeat/f-error-message": "0.3.1",
+    "@justeat/f-form-field": "1.1.0",
+    "@justeat/f-vue-icons": "1.10.0"
```

Comment:
Can these be put in at the top of `devDependencies`, to keep subsequent PRs clean, as they'll be reordered to there next time someone adds a package via Yarn